yaksh/views.py

```diff
@@ -281,7 +284,11 @@ def prof_manage(request):
 rights/permissions and log in."""
     user = request.user
     if user.is_authenticated() and is_moderator(user):
-        question_papers = QuestionPaper.objects.filter(quiz__course__creator=user)
+        question_papers = QuestionPaper.objects.filter(quiz__course__creator=user,
+                                                       quiz__is_trial=False
+                                                       )
+        trial_course = Course.objects.delete_all_trial_courses(user)
```

Comment:
Is it better to give manual delete?
Because then the moderator can see the attempt as well.
Also, I guess in the user mode, moderator can attempt infinite time even though number of attempt is set to one, because previous attempt is deleted!